manifests/f/FlightGear/FlightGear/2020.3.9/FlightGear.FlightGear.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: FlightGear.FlightGear
+PackageVersion: 2020.3.9
+PackageLocale: en-US
+Publisher: FlightGear
+PackageName: FlightGear Flight Simulator

```

```suggestion
Publisher: The FlightGear Team
PackageName: FlightGear
```
Quick little change to match ARP